addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4571,6 +4572,14 @@ registry.sizing = SnippetOptionWidget.extend({
                 return;
             }
 
+            // Locking the mutex during the resize. Started here to avoid
+            // empty returns.
+            const prom = new Promise(resolve => resizeResolve = () => resolve());
+            self.trigger_up("snippet_edition_request", { exec: () => {
+                self.trigger_up("disable_loading_effect");
```

Comment:
Are not you supposed to call `enable_loading_effect` at some point if you use that?